cmd/version/version.go

```diff
@@ -78,8 +84,64 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			getCliLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func getCliLatestVersion(currVersion string) {
+	// getting the latest version of openebsctl from sigs.k8s.io/krew-index
+	resp, err := http.Get("https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml")
+	util.CheckErrDefault(err, "Error reading response body")
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	util.CheckErrDefault(err, "Error reading response body")
+
+	var data map[string]interface{}
+	err = yaml.Unmarshal(body, &data)
+	util.CheckErrDefault(err, "Error parsing yaml")
+
+	latestVersion := data["spec"].(map[string]interface{})["version"].(string)
+	if !isLatestVersion(currVersion, latestVersion) {
+		fmt.Printf("\nYou are using older version (%s) of cli, latest available version is %s\n", currVersion, latestVersion)
+	}
+}
+
+func isLatestVersion(currVersion string, latestVersion string) bool {
```

Comment:
How about just returning `currVersion == latestVersion`, it'd work just about right as latestVersion would be updated via a GitHub action from now on

Reply:
actually right, this is great.